README.md

```diff
@@ -1,8 +1,6 @@
-<p><img src=".github/header.png?sanitize=true&raw=true" /></p>
+![auth0-php](.github/header.png)
```

Comment:
I uploaded this to the CDN so that it works across any site that might embed the readme

```suggestion
![auth0-php](https://cdn.auth0.com/website/sdks/banners/auth0-php-banner.png)
```